TestHelpers.Tests/MockFileSystemTests.cs

```diff
@@ -153,16 +153,16 @@ public void MockFileSystem_AddFile_ShouldMatchCapitalization_PerfectMatch()
             fileSystem.AddDirectory(MockUnixSupport.Path(@"C:\LOUD"));
 
             // Act
-            fileSystem.AddFile(@"C:\test\file.txt", "foo");
-            fileSystem.AddFile(@"C:\LOUD\file.txt", "foo");
-            fileSystem.AddDirectory(@"C:\test\SUBDirectory");
-            fileSystem.AddDirectory(@"C:\LOUD\SUBDirectory");
+            fileSystem.AddFile(MockUnixSupport.Path(@"C:\test\file.txt"), "foo");
```

Comment:
Should we, for consistency, use XFS as the namespace instead of MockUnixSupport? My first gut reaction was what is the difference between MockUnitSupport and XFS? If we chose one and stuck with it, I think the experience would be a little better.